### ISSUE TITLE
Styles and accessibility enhancement for the list selector

### ISSUE THREAD

**Issue by ktp242** [NYPL/staff-picks#182]
Styles and accessibility enhancement for the list selector.

It should have proper styles and layouts. The design can be found here
https://sketch.cloud/s/J88L0/all/staff-picks-v2/mobile-portrait-init-v1

Also, it should have proper functions for accessibility. This includes correct tabbing order, focusing on the title after selecting filters and lists, and voice over contents.
